build/maistra-release.sh

```diff
@@ -0,0 +1,62 @@
+set -ex
+
+if [ -n "${PULL_PULL_SHA}" ]; then
+    export ISTIO_OPERATOR_COMMIT=${PULL_PULL_SHA}
+else
+    export ISTIO_OPERATOR_COMMIT=$(git rev-parse HEAD)
+fi
+
+if [ -n "${PULL_NUMBER}" ]; then
+    export ISTIO_OPERATOR_PR=${PULL_NUMBER}
+fi
+
+export HUB=${HUB:-quay.io/maistra-dev}
+export REPO=https://copr.fedorainfracloud.org/coprs/g/maistra-dev/istio/repo/epel-8/group_maistra-dev-istio-epel-8.repo
+export VERSION=1.2.${ISTIO_OPERATOR_COMMIT}
+
+WORKDIR=$(mktemp -d)
+echo "Created temp directory ${WORKDIR}"
+
+function cleanup() {
+    echo "Cleaning up..."
+    popd
+    echo "Removing temp directory ${WORKDIR}"
+    rm -rf ${WORKDIR}
+}
+
+pushd ${WORKDIR}
+
+trap cleanup EXIT
+
+# build RPMs
+
+git clone https://github.com/Maistra/rpms.git
```

Comment:
I think Jonh is working on breaking the rpms repo apart.

Reply:
Yes, that's coming down the line